### PR TITLE
[SPARK-48846][PYTHON][DOCS] Fix the incorrect naming and missing params in func docs in `builtin.py`

### PR DESCRIPTION
**Pull request by wayneguow**
### What changes were proposed in this pull request?

Fix the incorrect naming and missing params in func docs in `builtin.py`.

### Why are the changes needed?

Some params' name in `pySpark` docs are wrong, for example:
image

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Passed GA.

### Was this patch authored or co-authored using generative AI tooling?

No.

### CODE REVIEW

**Comment by wayneguow** [comment]
cc @HyukjinKwon  @allisonwang-db

File: python/pyspark/sql/functions/builtin.py [hunk 1]
```diff
@@ -5494,7 +5494,7 @@ def first(col: "ColumnOrName", ignorenulls: bool = False) -> Column:
     |  Bob|         5|
     +-----+----------+
 
-    Now, to ignore any nulls we needs to set ``ignorenulls`` to `True`
+    Now, to ignore any nulls we need to set ``ignorenulls`` to `True`
```
Comment (allisonwang-db) [inline python/pyspark/sql/functions/builtin.py:5497]: nit: 
```suggestion
    To ignore any null values, set ``ignorenulls`` to `True`
```

**Review by allisonwang-db — COMMENTED** [review]
Thanks for the fix!

File: python/pyspark/sql/functions/builtin.py (hunk 1 above)
Comment (wayneguow) [reply python/pyspark/sql/functions/builtin.py]: Also updated for line 5770. Thank you for reviewing. @allisonwang-db

**Review by HyukjinKwon — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
Let's file a new JIRA next time :-).

**Comment by wayneguow** [comment]
> Let's file a new JIRA next time :-).

My fault, I probably copied the wrong id. I will definitely pay attention next time. When you have time, please help update the information of the right issue.
https://issues.apache.org/jira/browse/SPARK-48846 @HyukjinKwon Thanks.

**Pull request closed without merging**